src/crimson/os/seastore/btree/fixed_kv_node.h

```diff
@@ -109,41 +526,156 @@ struct FixedKVInternalNode
 	    ? &delta_buffer : nullptr;
   }
 
-  CachedExtentRef duplicate_for_write(Transaction&) override {
+  CachedExtentRef duplicate_for_write(Transaction& t) override {
+    LOG_PREFIX(FixedKVInternalNode::duplicate_for_write);
     assert(delta_buffer.empty());
-    return CachedExtentRef(new node_type_t(*this));
+    auto ext = new node_type_t(*this);
+    auto children = this->child_trans_views.template remove_trans_view<base_t>(t);
+    for (auto [child, pos] : children) {
+      auto &tracker = ext->child_trackers[pos];
+      tracker = new child_tracker_t(child);
+      t.new_pending_trackers.push_back(tracker);
+      t.trackers_to_rm.push_back(this->child_trackers[pos]);
```

Comment:
Does the same child exists in both or either of `this->child_trackers` and `this->child_trans_views`?

Reply:
For "stable" parents, their "stable" children exist in `this->child_trackers`, while their pending children exist in `this->child_trans_views`'.
For pending parents, all the children, pending or not, exist in `this->child_trackers`.

Reply:
Will add comments for this.